src/dso_api/dynamic_api/remote/views.py

```diff
@@ -83,14 +86,44 @@ def list(self, request, *args, **kwargs):
         """The GET request for listings"""
         data = self._call_remote(query_params=request.query_params)
         if "_embedded" in data and isinstance(data["_embedded"], dict):
-            data = next(iter(data["_embedded"].values())) if data["_embedded"] else []
-        serializer = self.get_serializer(data=data, many=True)
-        self.validate(serializer, data)
+            if data["_embedded"]:
+                ldata = None
+                for k, v in data["_embedded"].items():
+                    if not ldata:
+                        ldata = v
+                        if (
+                            self.dataset_id == "hcbrk"
+                            and self.table_id == "kadasternatuurlijkpersonen"
+                            and k.lower() == "kadasternatuurlijkpersonen"
+                        ):
+                            new_ldata = []
+                            for element in ldata:
+                                new_ldata.append(
+                                    self.filter_sensitive_data(request, element)
+                                )
+                            data["_embedded"][k] = new_ldata
```

Comment:
Dit gaat straks stuk als we responses streaming maken. Kan er niet op serializer niveau iets gedaan worden? Nu wordt er middleware-achtig naderhand wat aangepast.